packages/kbn-unified-data-table/src/utils/get_render_cell_value.tsx

```diff
@@ -109,6 +91,26 @@ export const getRenderCellValueFn = ({
       return <span className={CELL_CLASS}>-</span>;
     }
 
+    if (!!externalCustomRenderers && !!externalCustomRenderers[columnId]) {
```

Comment:
2 changes were done here
1. Passed extra params to custom renderer
2. Moved the if condition after all the hooks have been called. This was anti-pattern but since its a function and not a React component, the linting is not catching this.